fw-child/resources/js/child-functions.js

```diff
@@ -268,7 +281,11 @@ const $ = jQuery;
           pinned_item = new $.Zebra_Pin($('#control-bar-tabs'), {
             top_spacing: scroll_offset,
             contain: true,
+            class_name: 'tab-drawer-tabs--fixed',
           });
+
+          $(window).on('scroll resize', toggleClassOnControlbarAbsolute);
```

Comment:
If we scroll the page and reload it, the page loads with the content already scrolled. This callback may not be executed so the hamburger menu doesn't switch color from white to black, so it shows white on white (if the background of the page is white).

Reply:
You're right about the color issue on page load. I'll correct it. Although, this line does not control the color of the burger menu.